src/backend/distributed/master/master_modify_multiple_shards.c

```diff
@@ -266,81 +265,22 @@ static HTAB *
 OpenConnectionsToAllShardPlacements(List *shardIntervalList)
```

Comment:
You are right, they were almost identical. After the change of sending BEGIN after opening the connections, it is also renamed to `OpenTransactionsToAllShardPlacements`